src/System.Private.ServiceModel/tests/Common/Infrastructure/testproperties.props

```diff
@@ -150,6 +150,10 @@
     <PropertyGroup Condition="'$(SSL_Available)' == ''">
       <SSL_Available></SSL_Available>
     </PropertyGroup>
+
+  <PropertyGroup Condition="'$(IncludeTestsWithIssues)' == '' and $(WithCategories.ToLower().Contains('failing'))">
+    <IncludeTestsWithIssues>all</IncludeTestsWithIssues>
```

Comment:
Agreed. The name sounds Boolean.  And 'false' is also acceptable and treated the same as the default (i.e. don't include any tests with Issue)